specs/MultiProfile.md

```diff
@@ -22,6 +22,12 @@ multiple WebView2s running with separate profiles under a single user data direc
 browser instance at runtime), which means separate cookies, user preference settings, and various
 data storage etc., to help you build a more wonderful experience for your application.
 
+In order to manage cookies through profile, we're adding a get_CookieManager interface into profile.
+Users can use this interface to get the cooke manager, which is shared by all webview2s associated with
+this profile. Currently the cookie manager is got from WebView2 (see ICoreWebView2_2.get_CookieManager),
+as a standalone profile object can not support cooke management APIs. We will make cookie management
```

Comment:
"as" is confusing here. It sounds like you're saying "You get the cookie manager from the WebView2 as a standalone profile object". Probably should change to "because".

Reply:
Removed this sentence, because it describes the implementation internally.